utils/logalloc.cc

```diff
@@ -2193,9 +2193,11 @@ class reclaim_timer {
         auto time_level = _stall_detected ? log_level::warn : log_level::debug;
         auto info_level = _stall_detected ? log_level::info : log_level::debug;
         auto MiB = 1024*1024;
+        auto msg_extra = _stall_detected ? fmt::format(", at {}", current_backtrace()) : "";
```

Comment:
Generating a backtrace also takes time, so we're making the problem bigger. Oh well.

Reply:
Hopefully fixing the issue will provide big enough payback for the investment.
Maybe we can generate and print the backtrace only is the stall is longer than the previous maximum?

Reply:
yes

Reply:
I'll have the timeout back-off within a `reclaim_timer` hierarchy.  not sure if making it back-off globally is a good idea